test-validator/src/lib.rs

```diff
@@ -107,9 +107,14 @@ pub struct TestValidatorGenesis {
     pub max_genesis_archive_unpacked_size: Option<u64>,
     pub accountsdb_plugin_config_files: Option<Vec<PathBuf>>,
     pub accounts_db_caching_enabled: bool,
+    inactivate_feature_list: Option<Vec<Pubkey>>,
```

Comment:
Should be fine to remove the Option here:
```suggestion
    inactivate_feature_list: Vec<Pubkey>,
```

Reply:
Used `HashSet<Pubkey>` vs. `Vec<Pubkey>` in case called with same slice. Tested and verified in fb8b5b5